safehttp/mux.go

```diff
@@ -183,15 +184,17 @@ func (m methodHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 // interceptors.
 type handlerWithInterceptors struct {
 	handler   Handler
-	interceps []appliedInterceptor
+	interceps []AppliedInterceptor
 	disp      Dispatcher
 }
 
 // ServeHTTP calls the Before method of all the interceptors and then calls the
-// underlying handler.
+// underlying handler. Any panics that occur during the Before phase, during handler
+// execution or during the Commit phase are recovered here and a 500 Internal Server
```

Comment:
Recovering panics is a design decision that the writers of `net/http` regretted.

Wouldn’t it be safer to not make that mistake again?

Even if the answer is no, the comment should be updated to explain.

Reply:
Finding a better alternative to panics is one of our TODOs so they will disappear in v0 :)